tests/test_cases/test_array/test_array.py

```diff
@@ -351,6 +351,8 @@ async def test_discover_all(dut):
             pass_total = 1006
         else:
             pass_total = 1038
+    elif cocotb.LANGUAGE in ["verilog"] and cocotb.SIM_NAME.lower().startswith(("chronologic simulation vcs")):
```

Comment:
What about VHDL? Does it not run? Is it the same as the else clause?

Reply:
We don't support VHDL in VCS (it has a strange VHPI implementation).